src/staff/types.ts

```diff
@@ -0,0 +1,8 @@
+import { StaffMemberDetailsFragment } from "@saleor/graphql";
+
+type WithOptional<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>;
```

Comment:
do we have place for such utils in our codebase? I imagine that we would like to reuse this type in the future 🤔

Reply:
Removed 👌